src/sentry/tasks/derive_code_mappings.py

```diff
@@ -17,42 +24,114 @@
 logger = logging.getLogger("sentry.tasks.derive_code_mappings")
 
 
+@instrumented_task(
+    name="sentry.tasks.derive_code_mappings.process_organizations",
+    queue="derive_code_mappings",
+    max_retries=0,
+)  # type: ignore
+def derive_missing_codemappings(dry_run=False) -> None:
+    """
+    Queue tasks to process each organization to derive missing codemappings.
+    """
+    organizations = Organization.objects.filter(status=OrganizationStatus.ACTIVE)
+    for _, organization in enumerate(
+        RangeQuerySetWrapper(organizations, step=1000, result_value_getter=lambda item: item.id)
+    ):
+        if not features.has("organizations:derive-code-mappings", organization):
+            continue
+
+        # Create a celery task per organization
+        project_stacktrace_paths = identify_stacktrace_paths.delay(organization.id, dry_run=dry_run)
+        if not project_stacktrace_paths:
+            continue
+
+        integration = None
+        try:
+            integration = Integration.objects.filter(
+                organizations=organization.id,
+                provider="github",
+            )
+        except Integration.DoesNotExist:
+            logger.exception(f"Github integration not found for {organization.id}")
+
+        if integration is None or not integration.exists():
+            continue
+
+        integration = integration.first()
+        organization_integration = OrganizationIntegration.objects.filter(
+            organization=organization, integration=integration
+        )
+        if not organization_integration.exists():
+            continue
+
+        organization_integration = organization_integration.first()
+        install = integration.get_installation(organization.id)
+        trees: JSONData = install.get_trees_for_org()
+        for project, stacktrace_paths in project_stacktrace_paths.items():
```

Comment:
```suggestion
        trees_helper = CodeMappingTreesHelper(trees)
        for project, stacktrace_paths in project_stacktrace_paths.items():
```
I'm going to get rid of the function `derive_code_mappings` on a different PR.